model/http-common.yaml

```diff
@@ -52,6 +52,8 @@ groups:
         requirement_level: required
         brief: >
           Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+      - ref: http.route
```

Comment:
The brief and note on the registry attribute talk about server framework, could you please update them here?